docs/_src/language/explore.md

```diff
@@ -1,27 +1,196 @@
-# Explore Definition
+# Explores
 
-A shape is a data source and a list of additional properties which
-extend the original source in preparation for a
-passing it to a transformation stage.
+An explore consists of a data source along with a list of
+additional properties which exptend it, like adding fields
+or specifying joins. When an explore is used in a query,
+everything from the original source and all extensions can be used in queries and field definitions.
 
-* explore: _source_ fields _joins_
+```malloy
+define my_explore is (explore source
+  extensions
```

Comment:
The point is to gently introduce the syntax.

Maybe 

```malloy
define my_explore is (explore source
  -- Extensions go here
);
```

Reply:
maybe it'd be less confusing to use a real table rather than "my explore" and "source" as well, to show the basic stub? I tend to find those placeholder words confusing

Reply:
Nah, you're right. I'll delete. You make a good point about the placeholder words, and then if I use an actual example, it doesn't add anything to the example two inches below!